frontend/app/assets/javascripts/spree/frontend/views/spree/products/cart_form.js

```diff
@@ -263,6 +263,7 @@ Spree.ready(function($) {
           ), Spree.variantById($cartForm, variantId))
         },
         function(_error) {
+          document.querySelector('#no-product-available .no-product-available-text').innerText = _error
```

Comment:
@aplegatt let's add a check if `_error` isn't blank - this can happen (backend returning only 422 HTTP status without any error message)